Taskfile.yml

```diff
@@ -7,16 +7,19 @@ tasks:
 
   default:
     deps: [test]
-  
+
   install-deps:
     desc: Install tool dependencies.
-    deps:
-      - install-lint
+    cmds:
+      - go install github.com/golangci/golangci-lint/cmd/golangci-lint@{{.GOLANGCI_VERSION}}
+      - go install gotest.tools/gotestsum@{{.GOTESTSUM_VERSION}}
+      - go clean -cache -modcache
```

Comment:
This command is wrong in this context. It makes sense only when building a Docker image, here it will cause additional compilation when running the lint target.

 You can see the latest PR of Cogito for guidance if needed.

Reply:
Removed as suggested.